### PR TITLE
[PLAT-1181] Handle long press in the scene tree

### PR DESCRIPTION
**Pull request by amvertex**
## Summary
We want to handle long presses in the scene tree on mobile separately from single tap events. As a result, long pressing on a node in the scene tree will no longer select the item.

## Test Plan
Verify that long pressing on a node in the scene tree does not select the item.

## Release Notes
None

## Possible Regressions
Long presses in the scene tree

## Dependencies
None

### CODE REVIEW

File: packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx [hunk 1]
```diff
@@ -324,4 +339,18 @@ export class SceneTreeTableCell {
   ): void => {
     tree.toggleExpandItem(node);
   };
+
+  private clearLongPressTimer(): void {
```
Comment (MadisonEhlers-Vertex) [inline packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx:343]: Should we invoke the `clearLongPressTimer()` function on the disconnected callback?

**Review by MadisonEhlers-Vertex — APPROVED** [review]
Lgtm, left one suggestion

File: packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx (hunk 1 above)
Comment (amvertex) [reply packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx:344]: Good idea! Added.

**Review by MadisonEhlers-Vertex — APPROVED** [review]
I think this makes sense. One question about this is if we should handle a longPress event emit from the cell? Just a thought. That way we can use the tap event emitted from here.

**Pull request merged**